Car_owners/tests/test_views.py

```diff
@@ -0,0 +1,269 @@
+import datetime
+import pytest
+from rest_framework import status
+from rest_framework.test import APIClient
+from application.views import OwnerViewSet, CarViewSet
+
+
+@pytest.fixture
+def api_client() -> APIClient:
+    yield APIClient()
+
+
+@pytest.fixture
+def owner_data() -> dict[str, str]:
+    owner_data = {"name": "Andrzej", "surname": "Starczyk", "phone": "575849675"}
+    return owner_data
+
+
+@pytest.fixture
+def car_data(api_client: APIClient, owner_data: dict[str, str]) -> dict[str, str | int]:
```

Comment:
I don't like `car_data` getting APIClient object as an argument, this fixture should return a django model, which doesn't know anything about API. You can just call another fixture here if you want, but I wonder if that's really necessary to get `id` in such way